Package.swift

```diff
@@ -18,10 +18,10 @@ let package = Package(
   ],
   targets: [
     .plugin(
-      name: "AirbnbSwiftFormatPlugin",
+      name: "AirbnbSwiftFormat",
       capability: .command(
         intent: .custom(
-          verb: "format",
+          verb: "airbnb-swift-format",
```

Comment:
When another package consumes this plugin (e.g. https://github.com/airbnb/lottie-ios/blob/97c05f8272fc8e3a2cefb3d41c6bcb90c6d28739/Rakefile#L108) it uses this name. Let's pick a more specific name to avoid collisions.